README.md

```diff
@@ -24,3 +24,9 @@ rspec
 ## Solr Home Directory
 
 `solr_configs/` is configured to be the Solr home directory. Each application's Solr core configuration is a subdirectory of `solr_configs/`. Relevant Solr analysis library packages can be found within each core's `lib/` directory.
+
+## Adding a new core
+
+In practice, the home directory is at `solr/data`. The deploy scripts are designed for updating, not adding, cores. It is likely that solr config management process will be a topic of discussion soon. For now, if you're adding a new core you must create a directory for it in `solr/data`, then symlink the conf folder to the deployed folder. Finally, create the new core via the api, e.g.:
+
+`curl -d "action=CREATE&name=my-new-core" -X POST http://localhost:8983/solr/admin/cores`
```

Comment:
Can we recommend using the `solr` command-line util instead of raw curl requests?  It should be something like `solr create_core my-new-core`

Reply:
Changed it. Why do we prefer the command line over curl?

Reply:
Mostly because it works even if Solr is running on a different port, but also because it's shorter.

Reply:
👍